MonitorControl/UI/zh-Hant-TW.lproj/Main.strings

```diff
@@ -239,7 +239,7 @@
 "ltL-gR-K3Z.title" = "控制的螢幕：";
 
 /* Class = "NSTextFieldCell"; title = "Control your external displays brightness, contrast and volume directly from your Mac, using menulet sliders or the keyboard, including native Apple keys."; ObjectID = "Mh5-1A-apq"; */
-"Mh5-1A-apq.title" = "Control your external displays brightness, contrast and volume directly from your Mac, using menulet sliders or the keyboard, including native Apple keys.";
+"Mh5-1A-apq.title" = "使用菜單上的滑桿或者鍵盤（支持Apple原生快捷鍵），直接從Mac控制外接螢幕的亮度、對比度和音量。";
```

Comment:
The translation in Traditional Chinese is actually erroneous and doesn't follow the convention of the language. Please do not simply convert the characters from Simplified Chinese to Traditional Chinese. As the practice can cause confusion to both Traditional Chinese users and translators.

Reply:
I apologize for that. I noticed that some sentences weren't localized in Traditional Chinese, so I translated them together.

Reply:
Thanks for helping out. Feel free to tag me or assign me next time!